packages/comscore-analytics/types.ts

```diff
@@ -14,20 +14,51 @@ declare global {
   }
 }
 
-interface ComscoreAnalytics extends Analytics {
+/**
+ * `@frontity/comscore-analytics` package.
+ */
+interface ComscoreAnalytics extends Package {
   roots: Analytics["roots"] & {
     comscoreAnalytics: ReactType;
   };
   state: Analytics["state"] & {
+    /**
+     * State properties for the Comscore analytics package.
+     */
     comscoreAnalytics: {
+      /**
+       * Array of tracking ids associated to a Comscore client.
+       */
       trackingIds: string[];
     };
+    analytics: {
+      pageviews: {
+        /**
+         * Enable or disable sending pageviews to Comscore (default is `true`).
```

Comment:
You can use `@default` in TSDocs:

```ts
/**
 * Enable or disable sending pageviews to Comscore.
 *
 * @default true
 */
```

Reply:
Is it possible that the correct tag is `@defaultValue` instead of `@default`?

https://github.com/microsoft/tsdoc/blob/master/tsdoc/src/details/StandardTags.ts#L70

Reply:
Sure.

Reply:
Confirmed, let's use `@defaultValue`.